pkg/api/suggestion/v1alpha3/suggestion.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+package api.v1.alpha3;
+
+import "google/api/annotations.proto";
+
+service Suggestion {
+    rpc GetSuggestions(GetSuggestionsRequest) returns (GetSuggestionsReply);
+}
+
+message GetSuggestionsRequest {
+    Experiment experiment = 1;
+    repeated Trial trials = 2; // all completed trials owned by the experiment.
+    int32 request_number = 3;
+}
+
+message GetSuggestionsReply {
+    repeated Trial trials = 1; // trials should be created in the next run.
+}
+
+message Experiment {
+    string name = 1;
+    ExperimentSpec experiment_spec = 2;
+}
+
+message ExperimentSpec {
```

Comment:
You don't want to add `NasConfig` in `ExperimentSpec` on that step of implementation?

Reply:
This PR will add it, I think. It is still WIP.

Reply:
Cool, thank you!